Documentation/ceph-cluster-crd.md

```diff
@@ -617,8 +617,9 @@ Currently three health checks are implemented:
 * `osd`: health check on the ceph osds
 * `status`: ceph health status check, periodically check the Ceph health state and reflects it in the CephCluster CR status field.
 
-The liveness probe of each daemon can also be controlled via `livenessProbe`, the setting is valid for `mon`, `mgr` and `osd`.
-Here is a complete example for both `daemonHealth` and `livenessProbe`:
+The liveness probe and startup probe of each daemon can also be controlled via `livenessProbe` and
+`startupProbe` respectively. The settings are valid for `mon`, `mgr` and `osd`.
+Here is a complete example for both `daemonHealth`, `livenessProbe`, and `startupProbe`:
```

Comment:
So the code actually allows overriding the probes for `mds` as well, but this should really come from the CephFilesystem CR. Should we add that in a new PR?

Reply:
Agreed, it should be moved out so we can configure it from the CephFilesystem CR, just like the CephObjectStore does. I'm fine with a followup PR.